Formula/p/pango.rb

```diff
@@ -1,8 +1,9 @@
 class Pango < Formula
   desc "Framework for layout and rendering of i18n text"
-  homepage "https://pango.gnome.org"
-  url "https://download.gnome.org/sources/pango/1.54/pango-1.54.0.tar.xz"
-  sha256 "8a9eed75021ee734d7fc0fdf3a65c3bba51dfefe4ae51a9b414a60c70b2d1ed8"
+  # `https://pango.gnome.org/` is not accessible, upstream report, https://gitlab.gnome.org/GNOME/pango/-/issues/830
+  homepage "https://gitlab.gnome.org/GNOME/pango"
+  url "https://download.gnome.org/sources/pango/1.55/pango-1.55.0.tar.xz"
```

Comment:
upstream suggested to use `https://www.gtk.org/docs/architecture/pango` (redirected from `https://pango.org/`), which can be done as followup